src/container/MovieContainer.js

```diff
@@ -16,11 +17,11 @@ const MovieContainer = ({ movies, filterMovies, isLoaded }) => {
   })
 
   if (isLoaded) {
-    document.querySelector('.search-bar').classList.remove('hidden')
     return (
       <div>
+        <Form className="search-bar" filterMovies={filterMovies} rememberSearchQuery={rememberSearchQuery} />
         <div className='movie-container'>
-          {moviePosters.length ? moviePosters : <h2>No results</h2>}
+          {moviePosters.length ? moviePosters : <h2>There are no matches for your search. Try again?</h2>}
```

Comment:
We'll have to update the tests to account for this language change, but that's to be expected.